src/lib/dev-environment/dev-environment-cli.ts

```diff
@@ -998,38 +999,102 @@ export function getEnvTrackingInfo( slug: string ): Record< string, unknown > {
 	}
 }
 
+type EditorType = 'vscode' | 'cursor' | 'phpstorm';
+
+interface WorkspaceConfig {
+	extension: string;
+	generator: ( slug: string ) => string; // Returns path to generated workspace file
+}
+
+interface EditorConfig {
+	displayName: string;
+	candidates: string[];
+	workspace: WorkspaceConfig;
+}
+
+// Map of supported editors and their configurations
+const SUPPORTED_EDITORS: Record< EditorType, EditorConfig > = {
+	vscode: {
+		displayName: 'VS Code',
+		candidates: [ 'code', 'code-insiders', 'codium' ],
+		workspace: {
+			extension: 'code-workspace',
+			generator: generateVSCodeWorkspace,
+		},
+	},
+	cursor: {
+		displayName: 'Cursor',
+		candidates: [ 'cursor' ],
+		workspace: {
+			extension: 'code-workspace',
+			generator: generateVSCodeWorkspace,
+		},
+	},
+	phpstorm: {
+		displayName: 'PHPStorm',
+		candidates: [ 'phpstorm', 'pstorm' ],
```

Comment:
So this doesn't work, need to figure it out how to spawn a project with PHPStorm in CLI or change instructions on how to open the project.